CMakeLists.txt

```diff
@@ -93,19 +93,21 @@ macro(build_libyaml)
 endmacro()
 
 # Skip building yaml if the expected version is already present in the system
-if (yaml_FOUND)
+if(yaml_FOUND)
   if("${yaml_VERSION}" VERSION_EQUAL 0.2.5)
     set(_SKIP_YAML_BUILD 1)
   else()
-    message(WARNING "A wrong version of libyaml is already present in the system: ${yaml_VERSION}")
+    message(WARNING
+      "A wrong version of libyaml is already present in the system: ${yaml_VERSION}."
+      "It will be ignored and the expected version will be built.")
```

Comment:
@alsora nit: let's mention what the expected version is.

Reply:
I hard-coded a 0.2.5 in the message.
The number is used all over the file, but I think that using a variable would make the CMakelists much less readable.